src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -225,6 +227,20 @@ public static String parseNoteTags(String tags) throws ParseException {
         return trimmedTags;
     }
 
+    /**
+     * Parse a {@code String mappingMiscInformation}.
+     * Leading and trailing whitespaces will be trimmed.
+     */
+    public static MappingMiscInformation parseMappingMiscInformation(String mappingMiscInformation)
+            throws ParseException {
+        requireNonNull(mappingMiscInformation);
+        String trimmedMappingMiscInformation = mappingMiscInformation.trim();
+        if (!MappingMiscInformation.isValidMappingMiscInformation(trimmedMappingMiscInformation)) {
+            throw new ParseException(MappingMiscInformation.MESSAGE_CONSTRAINTS);
+        }
+        return new MappingMiscInformation(trimmedMappingMiscInformation);
+    }
```

Comment:
Can I ask what happens if it is empty string ""?

Reply:
I think currently empty string won't be accepted, like for all parameters.

Empty string is not allowed currently for MappingMiscInformation.

Will make changes soon to allow optional parameter for this mapping purpose.